staff/sys/stdin2syslog

```diff
@@ -1,49 +1,34 @@
-#!/usr/bin/env python3
-"""Write stdin logs to OCF's remote syslog server.
+#!/bin/bash
+set -euo pipefail
 
-You can use arbitrary stream names (without configuring them before); each
-stream will show up at syslog:/var/log/remote/<stream>.
-"""
-import argparse
-import re
-import subprocess
-import sys
-
-
-VALID_STREAM_NAME = re.compile('^[a-z0-9\-_]+$')
-
-
-class LoggingFailure(Exception):
-    pass
-
-
-def write_line(stream, line):
-    """Write a line to a stream.
+if [ $# -lt 1 ] || [ "$1" = "-h" -o "$1" = "--help" ]; then
+    echo "Usage: $0 stream
 
-    This might make sense to move to ocflib some day.
-    """
-    if not VALID_STREAM_NAME.match(stream):
-        raise ValueError('Bad stream name: {}'.format(stream))
+Write stdin logs to OCF's remote syslog server.
 
-    ret = subprocess.call((
-        'logger', '-T', '-n', 'syslog', '-P', '514', '-t', stream, '--', line,
-    ))
-    if ret != 0:
-        raise LoggingFailure('logger exited nonzero: {}'.format(ret))
-
-
-def main(argv=None):
-    parser = argparse.ArgumentParser(
-        description=__doc__,
-        formatter_class=argparse.RawDescriptionHelpFormatter,
-    )
-    parser.add_argument('stream')
-    args = parser.parse_args(argv)
-
-    for line in sys.stdin:
-        line = line.rstrip('\n')
-        write_line(args.stream, line)
-
-
-if __name__ == '__main__':
-    exit(main())
+You can use arbitrary stream names (without configuring them before); each
+stream will show up at syslog:/var/log/remote/<stream>.
+"
+    exit 1
+fi
+
+stream="$1"
+
+grep -qE '^[a-z0-9_][-a-z0-9_]+$' <<< "$stream" || {
+    echo "$0: invalid stream name."
+    exit 2
+}
+
+# Sorry, but this was the quickest and easiest way to do it and it works well
+# enough for our purposes...
+if [ $(lsb_release -cs) != "jessie" ]; then
+    exec logger -T -n syslog -P 514 -t "$stream"
```

Comment:
i should have commented though, this is a great find

once we're fully on stretch this script will become trivial, which is lovely